<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog_lapeditor</class>
 <widget class="QDialog" name="Dialog_lapeditor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>270</width>
    <height>372</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>270</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>270</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Lap Editor</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>4</number>
   </property>
   <property name="leftMargin">
    <number>4</number>
   </property>
   <property name="topMargin">
    <number>4</number>
   </property>
   <property name="rightMargin">
    <number>4</number>
   </property>
   <property name="bottomMargin">
    <number>4</number>
   </property>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Lap Infos</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="7" column="1">
       <widget class="QSpinBox" name="spinBox_endtime">
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximum">
         <number>10000</number>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_edit">
        <property name="text">
         <string>Edit:</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Lap:</string>
        </property>
       </widget>
      </item>
      <item row="6" column="0">
       <widget class="QLabel" name="label_2">
        <property name="maximumSize">
         <size>
          <width>65</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>Start-Time:</string>
        </property>
       </widget>
      </item>
      <item row="7" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>End-Time:</string>
        </property>
       </widget>
      </item>
      <item row="9" column="0">
       <widget class="QLabel" name="label_distance">
        <property name="text">
         <string>Distance:</string>
        </property>
       </widget>
      </item>
      <item row="9" column="1">
       <widget class="QDoubleSpinBox" name="doubleSpinBox_distance">
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>0</height>
         </size>
        </property>
        <property name="decimals">
         <number>3</number>
        </property>
        <property name="maximum">
         <double>199.990000000000009</double>
        </property>
        <property name="singleStep">
         <double>0.010000000000000</double>
        </property>
       </widget>
      </item>
      <item row="6" column="1">
       <widget class="QSpinBox" name="spinBox_starttime">
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximum">
         <number>10000</number>
        </property>
       </widget>
      </item>
      <item row="2" column="1" colspan="3">
       <widget class="QComboBox" name="comboBox_edit">
        <property name="enabled">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="7" column="2">
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>- </string>
        </property>
       </widget>
      </item>
      <item row="6" column="2">
       <widget class="QLabel" name="label_5">
        <property name="maximumSize">
         <size>
          <width>25</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>- </string>
        </property>
       </widget>
      </item>
      <item row="0" column="0" rowspan="2" colspan="4">
       <widget class="QFrame" name="frame_2">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>10</height>
         </size>
        </property>
        <property name="frameShape">
         <enum>QFrame::Box</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Sunken</enum>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <property name="spacing">
          <number>4</number>
         </property>
         <property name="leftMargin">
          <number>5</number>
         </property>
         <property name="topMargin">
          <number>5</number>
         </property>
         <property name="rightMargin">
          <number>5</number>
         </property>
         <property name="bottomMargin">
          <number>5</number>
         </property>
         <item>
          <widget class="QRadioButton" name="radioButton_update">
           <property name="text">
            <string>Update</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="radioButton_add">
           <property name="text">
            <string>Add</string>
           </property>
           <property name="checked">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="radioButton_del">
           <property name="text">
            <string>Delete</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="4" column="1" colspan="3">
       <widget class="QLineEdit" name="lineEdit_newName"/>
      </item>
      <item row="9" column="2">
       <widget class="QLabel" name="label_km">
        <property name="text">
         <string>Km</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1" colspan="3">
       <widget class="QComboBox" name="comboBox_lap"/>
      </item>
      <item row="6" column="3">
       <widget class="QLabel" name="label_start">
        <property name="minimumSize">
         <size>
          <width>50</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string>-</string>
        </property>
       </widget>
      </item>
      <item row="7" column="3">
       <widget class="QLabel" name="label_end">
        <property name="minimumSize">
         <size>
          <width>30</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string>-</string>
        </property>
       </widget>
      </item>
      <item row="11" column="1">
       <widget class="QSpinBox" name="spinBox_strokes"/>
      </item>
      <item row="11" column="0">
       <widget class="QLabel" name="label_strokes">
        <property name="text">
         <string>Strokes:</string>
        </property>
       </widget>
      </item>
      <item row="8" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Duration:</string>
        </property>
       </widget>
      </item>
      <item row="8" column="1">
       <widget class="QTimeEdit" name="timeEdit_duration">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="displayFormat">
         <string>mm:ss</string>
        </property>
       </widget>
      </item>
      <item row="12" column="0">
       <widget class="QLabel" name="label_type">
        <property name="text">
         <string>Type:</string>
        </property>
       </widget>
      </item>
      <item row="12" column="1">
       <widget class="QComboBox" name="comboBox_type"/>
      </item>
      <item row="13" column="1">
       <widget class="QProgressBar" name="progressBar_updateInt">
        <property name="maximum">
         <number>1</number>
        </property>
        <property name="value">
         <number>0</number>
        </property>
        <property name="textVisible">
         <bool>false</bool>
        </property>
        <property name="format">
         <string notr="true"/>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="frame">
     <property name="frameShape">
      <enum>QFrame::Box</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Sunken</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="spacing">
       <number>4</number>
      </property>
      <property name="leftMargin">
       <number>4</number>
      </property>
      <property name="topMargin">
       <number>4</number>
      </property>
      <property name="rightMargin">
       <number>4</number>
      </property>
      <property name="bottomMargin">
       <number>4</number>
      </property>
      <item>
       <widget class="QPushButton" name="pushButton_ok">
        <property name="text">
         <string>OK</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_close">
        <property name="text">
         <string>Close</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>radioButton_update</tabstop>
  <tabstop>radioButton_add</tabstop>
  <tabstop>radioButton_del</tabstop>
  <tabstop>comboBox_edit</tabstop>
  <tabstop>comboBox_lap</tabstop>
  <tabstop>lineEdit_newName</tabstop>
  <tabstop>spinBox_starttime</tabstop>
  <tabstop>spinBox_endtime</tabstop>
  <tabstop>timeEdit_duration</tabstop>
  <tabstop>doubleSpinBox_distance</tabstop>
  <tabstop>spinBox_strokes</tabstop>
  <tabstop>pushButton_ok</tabstop>
  <tabstop>pushButton_close</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
